File: Day 5/loops_v2.cs
using System;
class Solution {
    static void Main(String[] args) {
        string ans = "";
        int numOfTests = Convert.ToInt32(Console.ReadLine());
        for (int i = 0; i < numOfTests; ++i) {
    	    string[] strIn = Console.ReadLine().Split(' ');
            int a = Convert.ToInt32(strIn[0]);
            int b = Convert.ToInt32(strIn[1]);
            int N = Convert.ToInt32(strIn[2]);
    	    int currentSum = a;
    	    for (int j = 0; j < N; ++j) {
    		    currentSum += Convert.ToInt32(Math.Pow(2, j)) * b;
    		    ans += currentSum.ToString() + " ";
    	    }
    	    ans += "\n";
        }
        Console.Write(ans);
    }
}

File: Day 5/loops_v2.scala
object Solution {
    def main(args: Array[String]) {
        var ans = ""
        val numOfTests = scala.io.StdIn.readInt()
        for (i <- 1 to numOfTests) {
            val Array(a, b, n) = scala.io.StdIn.readLine.split(" ").map(_.toInt)
            var currentSum = a
            for (j <- 0 to n - 1) {
                currentSum += scala.math.pow(2, j).toInt * b
                ans += currentSum.toString + " "
            }
            ans += "\n"
        }
        System.out.print(ans)
    }
}